### ISSUE TITLE
When adding multiple pads handle_pad_added/3 invokes with all pads already in context

### ISSUE THREAD

**Issue by Hajto** [membraneframework/membrane_core#258]
If we have an element with dynamic pads that have `handle_pad_added/3` implemented as follows:
```
@impl true
def handle_pad_added(_pad, _ctx, %State{work_state: work_state} = state) do
  IO.inspect(ctx)
  {:ok, state}
end
```

and we add multiple pads at once with spec action sent from the parent, then `IO.inspect(ctx)` will print a message as follows:

```
Invoked handle pad: %Membrane.Element.CallbackContext.PadAdded{
  pads: %{
    {Membrane.Pad, :output, 3} => %Membrane.Pad.Data{...},
    {Membrane.Pad, :output, 2} => %Membrane.Pad.Data{...},
    ...
  },
}
```

**Comment by mat-hek** [comment]
That should be probably solved together with #214

**Comment by mat-hek** [comment]
fixed by #313
